network-config-analyzer/NetworkConfig.py

```diff
@@ -259,7 +279,7 @@ def get_captured_pods(self):
         :rtype: Peer.PeerSet
         """
         captured_pods = Peer.PeerSet()
-        for policy in self.sorted_policies:
+        for policy in self.ingress_deny_policies + self.sorted_policies + self.ingress_allow_policies:
```

Comment:
Do we want Ingress policies to be considered here or not?

Reply:
Since we decided that Ingress may only deny some connections (but cannot add connections other than those defined by the network policy), it should not affect captured or affected pods